script/node/verify-exports.js

```diff
@@ -0,0 +1,157 @@
+// COPIED FROM BIGMAVEN
```

Comment:
@juanca This exercise was interesting .. maybe we should consider a first step for creating sharable linters is to port the helpers directory? Maybe the `helpers/directories.js` would become `helpers/bigmaven-directories.js` and so on. The only thing I really had to change here otherwise was the first dozen lines or so which also deal with assembling the directory paths.

I could see scripts like this being updated to take an optional configuration path that houses the list of directories to sweep, and another list of file types to work on, and anything else specific to the repo it's placed in. E.g.: 

```json
{
	"file_types": ["js", "jsx", "coffee"],
	"directories": [
		"example",
		"spec",
		"lib"
	]
}
```

Reply:
Each repository will most likely have a different structures thus I think each repository should keep their own scripts for pulling up appropriate files. The linter scripts, however, are pretty repository-agnostic and should take a list of files as part of their API -- this would pretty much mirror how ES lint does its processing too. 

Anyway, we can whiteboard it once we think we should start the story.